string map_name
string position_name
---
geometry_msgs/PoseStamped pose_stamped